mozilla/foundation.mozilla.org
# external links in rich text blocks now opening in new tab

Closes #8002 

**Link to sample test page**: https://foundation-s-rich-text--1wiazw.mofostaging.net/en/blog/initial-test-blog-post-with-fixed-title/

**Steps to test**:

1. Please visit the test page above 
2. In the posts content, please click on any of the external links such as the ones that say "This is a link to a fake URL!" or the ones found in the image or quote block.
3. These all should open to a new tab
4. If you would like to test another page type, visit https://foundation-s-rich-text--1wiazw.mofostaging.net/en/publication-page-with-child-article-pages/fixed-title-article-page/ to test out the external rich text links found there.
5. If everything is working as expected, testing is complete!
